exporters/metric/cortex/cortex.go

```diff
@@ -119,31 +122,52 @@ func (e *Exporter) ConvertToTimeSeries(checkpointSet export.CheckpointSet) ([]*p
 		// Convert based on aggregation type
 		agg := record.Aggregation()
 
-		// Check if aggregation has Sum value
-		if sum, ok := agg.(aggregation.Sum); ok {
+		// Check if aggregation has Histogram value
+		if histogram, ok := agg.(aggregation.Histogram); ok {
+			tSeries, err := convertFromHistogram(record, histogram)
+			if err != nil {
+				return err
+			}
+			timeSeries = append(timeSeries, tSeries...)
+			// Check if aggregation has sum value
+		} else if sum, ok := agg.(aggregation.Sum); ok {
 			tSeries, err := convertFromSum(record, sum)
 			if err != nil {
 				return err
 			}
 
 			timeSeries = append(timeSeries, tSeries)
-		}
 
-		// Check if aggregation has MinMaxSumCount value
-		if minMaxSumCount, ok := agg.(aggregation.MinMaxSumCount); ok {
-			tSeries, err := convertFromMinMaxSumCount(record, minMaxSumCount)
-			if err != nil {
-				return err
-			}
+			// Check if aggregation has MinMaxSumCount value
+			if minMaxSumCount, ok := agg.(aggregation.MinMaxSumCount); ok {
+				tSeries, err := convertFromMinMaxSumCount(record, minMaxSumCount)
+				if err != nil {
+					return err
+				}
 
-			timeSeries = append(timeSeries, tSeries...)
+				timeSeries = append(timeSeries, tSeries...)
+
+				// Check if aggregation has a Distribution value
+				if distribution, ok := agg.(aggregation.Distribution); ok && len(e.config.Quantiles) != 0 {
+					tSeries, err := convertFromDistribution(record, distribution, e.config.Quantiles)
+					if err != nil {
+						return err
+					}
+
+					timeSeries = append(timeSeries, tSeries...)
+				}
+			}
+			// Check if aggregation has lastValue
 		} else if lastValue, ok := agg.(aggregation.LastValue); ok {
 			tSeries, err := convertFromLastValue(record, lastValue)
 			if err != nil {
 				return err
 			}
 
 			timeSeries = append(timeSeries, tSeries)
+		} else {
+			// Report to the user when no conversion was found
+			log.Printf("No conversion found for record: %s\n", record.Descriptor().Name())
```

Comment:
Apologies if this has been previously addressed, but is the standard `log` package the common logging package for plugins in the SDK?

Reply:
Looking at it more closely, fmt is much more prevalent, so I'll switch to that.

Reply:
That sounds surprising to me. I'd double check maybe with someone more familiar with SDK of best logging practices. I'd expect `log` over `fmt`, but I was expecting potentially another logging package or framework to be utilized throughout the SDK. For example the [Uber zap logger](https://github.com/uber-go/zap) seems pretty standard in the collector-contrib repo.